test/Microsoft.ML.Tests/FeatureContributionTests.cs

```diff
@@ -71,9 +71,10 @@ public void TestFeatureImportance()
             var srcDV = bldr.GetDataView();
 
             var pipeline = ML.Transforms.Concatenate("Features", "X1", "X2Important", "X3", "X4Rand")
+                .AppendCacheCheckpoint(ML)
                 .Append(ML.Transforms.Normalize("Features"));
             var data = pipeline.Fit(srcDV).Transform(srcDV);
-            var model = ML.Regression.Trainers.OnlineGradientDescent().Fit(data);
```

Comment:
Because caching change the behavior of `batch size,` we replace `OnlineGradientDescent` with another linear regressor. This should not affect the goal of this test.